tias/src/Components/Scheduling/SchedulingBlock.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react'
+
+const colors = new Map()
+colors.set(121, '#0086B6');
+colors.set(221, '#434F6F');
+colors.set(312, 'white');
+colors.set(313, '#405AB9');
+colors.set(314, 'white');
+colors.set(315, '#009489');
```

Comment:
We'll want to fetch the list of available courses from an API query. As far as colors, we *could* do this in the database (affiliated with the course) or we could have it be separated into a config.